CHANGELOG.md

```diff
@@ -1,15 +1,53 @@
 # nf-core/tools: Changelog
 
+## v1.5dev
+
+#### Template pipeline
+* Dropped Singularity file
+* Summary now logs details of the cluster profile used if from [nf-core/configs](https://github.com/nf-core/configs)
+* Dockerhub is used in favor of Singularity Hub for pulling when using the Singularity profile
+* Changed default container tag from latest to dev
+* Brought the logo to life
+* Change the default filenames for the pipeline trace files
+* Remote fetch of nf-core/configs profiles fails gracefully if offline
+* Remove `process.container` and just directly define `process.container` now.
+
+#### Tools helper code
+* New `nf-core launch` command to interactively launch nf-core pipelines from command-line
+  * Works with a `parameters.settings.json` file shipped with each pipeline
+  * Discovers additional `params` from the pipeline dynamically
+* Drop Python 3.4 support
+* `nf-core list` now only shows a value for _"is local latest version"_ column if there is a local copy.
+* Lint markdown formatting in automated tests
+  * Added `markdownlint-cli` for checking Markdown syntax in pipelines and tools repo
+* Syncing now reads from a `blacklist.json` in order to exclude pipelines from being synced if necessary.
+* Added nf-core tools API description to assist developers with the classes and functions available.
+  * Docs are automatically built by Travis CI and updated on the nf-co.re website.
+* Introduced test for filtering remote workflows by keyword.
+* Build tools python API docs
+  * Use Travis job for api doc generation and publish
+* Bump `conda` to 4.5.12 in base nf-core Dockerfile
```

Comment:
```suggestion
* Bump `conda` to 4.5.12 in base nf-core Dockerfile
```

As listed in ''OTHER'' down further, we can drop this as we already updated to 4.6.X later 😆

Reply:
Fixed 👍